lib/modules/platform/codecommit/iam-client.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { PLATFORM_BAD_CREDENTIALS } from '../../../constants/error-messages';
+
+describe('modules/platform/codecommit/iam-client', () => {
+  let iamClient: any;
+  let iam: any;
+
+  beforeEach(() => {
+    jest.resetModules();
+    jest.mock('@aws-sdk/client-iam');
```

Comment:
use AWS mock instead. see S3 tests in util/s3

Reply:
tried, doesn't mock